doc/src/manual/running-external-programs.md

```diff
@@ -273,8 +273,8 @@ that shells cannot.
 
 Julia can run multiple commands in parallel:
 
-```julia-repl
-julia> run(`echo hello` & `echo world`)
+```jldoctest; filter = r"(world\nhello|hello\nworld)
```

Comment:
It is pretty cool that we can test examples like this now even though the order is undefined.